bot/exts/moderation/dm_relay.py

```diff
@@ -53,14 +53,14 @@ async def dmrelay(self, ctx: Context, user: discord.User, limit: int = 100) -> N
             f"User: {user} ({user.id})\n"
             f"Channel ID: {user.dm_channel.id}\n\n"
         )
+        file = PasteFile(content=metadata+output, lexer="text")
```

Comment:
Should have spaces around operators as per PEP 8
```suggestion
        file = PasteFile(content=metadata + output, lexer="text")
```